packages/calcite-components/src/utils/interactive.ts

```diff
@@ -104,3 +121,76 @@ export function updateHostInteraction(
 
   component.el.removeAttribute("aria-disabled");
 }
+
+function blockInteraction(component: InteractiveComponent): void {
+  component.el.click = interceptedClick;
```

Comment:
Is there any chance a user could already have a click property on an el?

Reply:
Only if they're also hijacking `HTMLElement.click` for some magical purposes. We can revisit this if users report odd behavior.